src/test/java/calculator/UserTest.java

```diff
@@ -0,0 +1,44 @@
+package calculator;
+
+import static org.assertj.core.api.Assertions.assertThatIllegalArgumentException;
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.io.PrintStream;
+import org.junit.jupiter.api.Test;
+import stringCalculator.User;
+
+public class UserTest {
+
+    @Test
+    void 공백존재() {
+
+        User user = new User();
+        String userInput = "1 + 2 *  4 - 1";
+        OutputStream out = new ByteArrayOutputStream();
+        System.setOut(new PrintStream(out));
+        InputStream in = new ByteArrayInputStream(userInput.getBytes());
+        System.setIn(in);
+        //user.getUserInput();
+
+        assertThatIllegalArgumentException()
+            .isThrownBy(user::getUserInput
+            );
+
+    }
+
+    void 공백입력() {
+        User user = new User();
+        String userInput = "";
+        OutputStream out = new ByteArrayOutputStream();
+        System.setOut(new PrintStream(out));
+        InputStream in = new ByteArrayInputStream(userInput.getBytes());
+        System.setIn(in);
+        //user.getUserInput();
+
+        assertThatIllegalArgumentException()
+            .isThrownBy(user::getUserInput
+            );
```

Comment:
아래와 같이 닫는 괄호를 같은 라인에 둬도 될 것 같은데, 닫는 괄호를 한 줄 띄우신 이유가 있을까요? 🤔
```suggestion
            .isThrownBy(user::getUserInput);
```